app/toolbar/toolbar.component.css

```diff
@@ -73,143 +65,33 @@ md-toolbar span a.topnav {
   color: #FF8200;
 }
 
-a.dropdown-toggle {
-  color: #FFFFFF;
-  margin: 0;
-  height: 100%;
-}
-
-a.dropdown-toggle md-icon {
-  vertical-align: middle;
-}
-
-a.dropdown-toggle span.profile-name {
-  vertical-align: middle;
-}
-
-a.dropdown-toggle:hover {
-  text-decoration: none;
-}
-
-.dropdown {
-  position: relative;
-  margin: 0;
-  height: 100%;
-}
-
-ul.dropdown-menu {
+ul.toolbar-menu {
```

Comment:
The menu should have a shadow, so the user can easily see what exactly had pop up.